src/hdx/utilities/loader.py

```diff
@@ -19,24 +19,38 @@ def load_text(
     encoding: str = "utf-8",
     strip: bool = False,
     replace_newlines: Optional[str] = None,
+    replace_line_separators: Optional[str] = None,
     loaderror_if_empty: bool = True,
+    default_line_separator: str = "\n",
 ) -> str:
     """Load file into a string removing newlines.
 
     Args:
         path (str): Path to file
         encoding (str): Encoding of file. Defaults to utf-8.
         strip (bool): Whether to strip whitespace from start and end. Defaults to False.
-        replace_newlines (Optional[str]): String with which tp replace newlines. Defaults to None (don't replace).
+        replace_newlines (Optional[str]): String with which to replace newlines. Defaults to None (don't replace). (deprecated 2024-02-07)
+        replace_line_separators (Optional[str]): String with which to replace newlines. Defaults to None (don't replace).
         loaderror_if_empty (bool): Whether to raise LoadError if file is empty. Default to True.
+        default_line_separator (str): line separator to be replaced if replace_newlines is True
```

Comment:
Description needs fixing eg. something like:
default_line_separator (str): line separator to be replaced if replace_line_separators is not None

Reply:
Update made, as requested - I also added a date to the depreciation warning